tests/unit/test_generate_ip_address_policy.py

```diff
@@ -0,0 +1,19 @@
+import restrict_download_region.restrict_region as restrict_region
+from pytest_mock import MockerFixture
+
+
+def test_generate_ip_address_policy(mocker: MockerFixture):
+    bucket_name = "mytestbucket"
+    # function under test
+
+    ip_prefixes = ['15.230.56.104/31', '2600:1f19:8000::/36']
+
+    assert restrict_region.generate_ip_address_policy(bucket_name, ip_prefixes)\
```

Comment:
I think  ip_prefixes can be None on a `Delete` event?  what happens in that case?

Reply:
Sorry about the confusion. The `None` possibility was introduced when I was still trying to handle multiple buckets, but didn't want to re-fetch the IP ranges for each bucket. I'll refactor the usage of this function